src/libraries/System.Private.CoreLib/src/System/DateTime.Windows.cs

```diff
@@ -41,26 +41,28 @@ public static unsafe DateTime UtcNow
         }
 
         [MethodImpl(MethodImplOptions.NoInlining)]
-        internal static unsafe bool IsValidTimeWithLeapSeconds(int year, int month, int day, int hour, int minute, DateTimeKind kind)
+        private static unsafe bool IsValidTimeWithLeapSeconds(DateTime value)
```

Comment:
This is likely a deoptimization. Decomposing DateTime has its overhead.

Reply:
This method is never called in practice. And hopefully there will be [no more leap seconds added in the future](https://en.wikipedia.org/wiki/Leap_second#Future_of_leap_seconds).